content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.Stop.tsx

```diff
@@ -33,16 +33,16 @@ const StandaloneTitle = styled(Space).attrs({
     props.theme.color(getTypeColor('captions-and-transcripts'))};
 `;
 
-type LevelProps = { level: number };
+type LevelProps = { $level: number };
```

Comment:
Should be [transient](https://styled-components.com/docs/api#transient-props) to avoid being passed in the DOM